src/main/java/org/candlepin/subscriptions/resource/InstancesResource.java

```diff
@@ -79,6 +80,17 @@ public class InstancesResource implements InstancesApi {
   private final HostRepository hostRepository;
   private final PageLinkCreator pageLinkCreator;
 
+  @SuppressWarnings("linelength")
+  public static final Map<HostReportSort, String> HOST_SORT_PARAM_MAPPING =
```

Comment:
Unneeded.

Reply:
This was needed by some tests in HostRepositoryTest.
However, these tests were verifying the sort feature from Spring, so I don't think we really need them. Just delete this HOST_SORT_PARAM_MAPPING field and these tests.